src/analytics/event.ts

```diff
@@ -9,6 +9,23 @@ export const event = {
   analyticsTrackingDisabled: 'analytics_tracking.disabled',
   analyticsTrackingEnabled: 'analytics_tracking.enabled',
   swapSubmitted: 'Submitted Swap',
+
+  // notification promo sheet was shown
+  notificationsPromoShown: 'notifications_promo.shown',
```

Comment:
curious why this pattern vs something more human readable

Reply:
Great question. At my previous jobs, the data science team always wanted to be able to filter by something consistent e.g. a category, kinda like your initial idea. So they would do queries for like "contains `notifications_promo`" in order to group things. It also removes some variables and adds consistency, terseness. Thoughts?